Makefile

```diff
@@ -19,17 +19,26 @@ iso_creation: init
 	# http://askubuntu.com/questions/153833/why-cant-i-mount-the-ubuntu-12-04-installer-isos-in-mac-os-x
 	# http://www.syslinux.org/wiki/index.php?title=Doc/isolinux#HYBRID_CD-ROM.2FHARD_DISK_MODE
 	dd if=/dev/zero bs=2k count=1 of=${BUILD_DIR}/tmp.iso
-	dd if=$(BUILD_DIR)/$(ISO_NAME) bs=2k skip=1 >> ${BUILD_DIR}/tmp.iso
-	mv -f ${BUILD_DIR}/tmp.iso $(BUILD_DIR)/$(ISO_NAME)
+	dd if=$(BUILD_DIR)/$(ISO_NAME).iso bs=2k skip=1 >> ${BUILD_DIR}/tmp.iso
+	mv -f ${BUILD_DIR}/tmp.iso $(BUILD_DIR)/$(ISO_NAME).iso
 
 .PHONY: centos_iso
 centos_iso: KICKSTART_FILE=centos-7-minimal.ks
 centos_iso: KICKSTART_TEMPLATE=centos-7-minimal.template
-centos_iso: ISO_NAME=minishift-centos.iso
+centos_iso: ISO_NAME=minishift-centos
 centos_iso: iso_creation
 
 .PHONY: rhel_iso
+ifndef rhel_tree_url
```

Comment:
There is some issue (regression) when running `make` which run centos-iso here. @praveenkumar is working on it.